aging_due_report/wizard/wizard.py

```diff
@@ -185,10 +184,7 @@ class AccountAgingWizardCurrency(models.TransientModel):
     _description = 'Account Aging Wizard Currency'
     _rec_name = 'currency_id'
 
-    @api.depends('partner_ids', 'partner_ids.residual',
-                 'partner_ids.not_due', 'partner_ids.span01',
-                 'partner_ids.span02', 'partner_ids.span03',
-                 'partner_ids.span04', 'partner_ids.span05')
+    @api.depends('aaw_id.user_id')  # /!\ NOTE: avoid multi-triggering
     def _get_amount(self):
```

Comment:
This is a global `get_amount` vs `get_amount` by line above.
And both are using the same operations but one is cumulative and other one is just by line.
What about use just `line._get_amount` and assign a cumulative sum to parent.

Something like as:
```python
_name = 'account.aging.wizard.partner'

def _get_amount(self):
    line['span01'] = 10
    line.parent_id['span01'] += line['span01']
```

With this feature you won't need to compute the same value 2 times to totalize and to get the line value.